### ISSUE TITLE
Изменить точку с запятой на крест

### ISSUE THREAD

**Issue by Sank-WoT** [KanatnikovMax/znanie-drevnix#28]
Мне непонятен символ точка с запятой? Очень похоже на символы ящерские. Предлагаю заменить на славянский крест.

**Comment by GulgDev** [comment]
Согласен. На самом деле Ящеры прячут своё потомство в этих символах (;) - сверху яйцо, а снизу уже хвост поганого Ящера виден!!

**Comment by TBoRCMepT** [comment]
А может лучше на твердый знак?

**Comment by xi816** [comment]
Русь::мловить << "Я рус"ъ

**Comment by its5Q** [comment]
> А может лучше на твердый знак?

Тогда не сможем Ъ использовать в именах
